docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -119,6 +119,9 @@ The source cloud always uses the same password for cells databases. For that rea
 +
 [source,yaml]
 ----
+ifeval::["{build_variant}" == "ospdo"]
+oc project <OSPDO_NAMESPACE>
```

Comment:
please align with tests. There is logic around NAMESPACE env var which we should for consistency

Reply:
ack